core/commands/diff.py

```diff
@@ -24,14 +24,19 @@
 class GsDiffCommand(WindowCommand, GitCommand):
 
     """
-    Create a new view to display the project's diff.  If `in_cached_mode` is set,
-    display a diff of the Git index.
+    Create a new view to display the difference of `target_commit`
+    against `base_commit`. If `target_commit` is None, compare
+    working directory with `base_commit`.  If `in_cached_mode` is set,
+    display a diff of the Git index. Set `disable_stage` to True to
+    disable Ctrl-Enter in the diff view.
     """
 
     def run(self, **kwargs):
         sublime.set_timeout_async(lambda: self.run_async(**kwargs), 0)
 
-    def run_async(self, in_cached_mode=False, file_path=None, current_file=False, base_commit=None):
+    def run_async(self, in_cached_mode=False, file_path=None,
+                  current_file=False, base_commit=None, target_commit=None, disable_stage=False,
+                  title=None):
```

Comment:
The line-breaks here suggest `title` is special in some way.  Since it can't fit on one line, would prefer either even distribution of args on each subsequent line, or a line break for every arg, even thought its verbose.